### PR TITLE
Removed duplicates and added missing lines

### PR DESCRIPTION
**Pull request by ianf77**
Removed duplicate hub parameters and added missing EDA parameters

Running the script as described fails

https://issues.redhat.com/browse/AAP-20952

### CODE REVIEW

File: downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc [hunk 1]
```diff
@@ -95,11 +95,10 @@ sso_keystore_password=''
...
-#
 # automationedacontroller_controller_verify_ssl = true
-
-# Certificate and key to install in Automation Hub node
-# automationhub_ssl_cert=/path/to/automationhub.cert
-# automationhub_ssl_key=/path/to/automationhub.key
+#
+# Certificate and key to install in Automation EDA Controller node
+# automationedacontroller_ssl_cert=/root/eda.crt
+# automationedacontroller_ssl_key=/root/eda.key
```
Comment (SNiemann15) [inline downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc:102]:
```suggestion
# automationedacontroller_ssl_cert=/path/to/eda.crt
# automationedacontroller_ssl_key=/path/to/eda.key
```

File: downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc [hunk 2]
```diff
@@ -95,11 +95,10 @@ sso_keystore_password=''
...
 # Boolean flag used to verify Automation Controller's
 # web certificates when making calls from Automation EDA Controller.
-#
 # automationedacontroller_controller_verify_ssl = true
-
-# Certificate and key to install in Automation Hub node
-# automationhub_ssl_cert=/path/to/automationhub.cert
-# automationhub_ssl_key=/path/to/automationhub.key
+#
+# Certificate and key to install in Automation EDA Controller node
```
Comment (EMcWhinn) [inline downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc:100]: Would it be worth adding the attribute here for the EDA product name consistency?

{EDAcontroller}

**Review by EMcWhinn — COMMENTED** [review]
Just one comment on attribute, ignore if not relevant

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Pull request merged**